xblockutils/studio_editable.py

```diff
@@ -89,10 +97,49 @@ def studio_view(self, context):
             if field_info is not None:
                 context["fields"].append(field_info)
         fragment.content = loader.render_template('templates/studio_edit.html', context)
+
+        fragment.add_css(pkg_resources.resource_string(__name__, 'public/tabs.css'))
         fragment.add_javascript(loader.load_unicode('public/studio_edit.js'))
+
         fragment.initialize_js('StudioEditableXBlockMixin')
         return fragment
 
+    def fetch_tabs(self, context=None):
+        """
+        Fetch and render the defined tabs' templates from the child XBlock.
```

Comment:
It's not a child XBlock. It's the same XBlock that includes this mixin. So maybe say "from this XBlock"